vyper/types.py

```diff
@@ -154,7 +154,7 @@ def canonicalize_type(t, is_indexed=False):
         return canonicalize_type(t.subtype) + "[%d]" % t.count
     if isinstance(t, TupleType):
         return "({})".format(
-            ",".join(canonicalize_type(x) for x in t.subtypes)
+            ",".join(canonicalize_type(x) for x in t.members)
```

Comment:
Was this a mistake in the code?

Reply:
Yes same copy-paste-from-StructType problem.